css/components/Alert/alert.css

```diff
@@ -0,0 +1,67 @@
+.alert {
+  border-radius: 0.5rem;
+  font-weight: bold;
+  margin: 1rem 0;
+  max-width: 25rem;
+  padding: 1rem;
+}
+
+.alert-f-danger {
```

Comment:
does f mean fill in `.alert-f-danger`?  Maybe you can use `alert-danger-solid` `alert-danger-outline`.

Reply:
Yeah, it means `fill`. Thanks, I will change them.